app/tests/Fixture/UsersFixture.php

```diff
@@ -22,8 +22,8 @@ public function init(): void
                 'id' => 1,
                 'email' => 'Lorem ipsum dolor sit amet',
                 'password' => 'Lorem ipsum dolor sit amet',
-                'created' => '2023-10-26 05:40:13',
-                'modified' => '2023-10-26 05:40:13',
+                'created' => '2023-10-27 04:51:46',
+                'modified' => '2023-10-27 04:51:46',
```

Comment:
Perhaps, you've run the bake command for the model within this PR. Therefore, it's updating the fixture by adjusting the date & time for fields created & modified.

From the bake command, we can defined the options `--help`. We can see the supported options. Eg:

```
docker compose exec php-app bin/cake bake model --help

Bake table and entity classes.

Usage:
cake bake model [options] [<name>]

Options:

--connection, -c           The datasource connection to get data from.
                           (default: default)
--display-field            The displayField if you would like to choose
                           one.
--fields                   A comma separated list of fields to make
                           accessible.
--force, -f                Force overwriting existing files without
                           prompting.
--help, -h                 Display this help.
--hidden                   A comma separated list of fields to hide.
--no-associations          Disable generating associations.
--no-entity                Disable generating an entity class.
--no-fields                Disable generating accessible fields in the
                           entity.
--no-fixture               Do not generate a test fixture skeleton.
--no-hidden                Disable generating hidden fields in the
                           entity.
--no-rules                 Disable generating a rules checker.
--no-table                 Disable generating a table class.
--no-test                  Do not generate a test case skeleton.
--no-validation            Disable generating validation rules.
--plugin, -p               Plugin to bake into.
--primary-key              The primary key if you would like to manually
                           set one. Can be a comma separated list if you
                           are using a composite primary key.
--quiet, -q                Enable quiet output.
--skip-relation-check      Generate relations for all "example_id"
                           fields without checking the database if a
                           table "examples" exists.
--table                    The table name to use if you have
                           non-conventional table names.
--theme, -t                The theme to use when baking code.
                           (choices: Bake|Migrations)
--update                   Update generated methods in existing files.
                           If the file doesn't exist it will be created.
--verbose, -v              Enable verbose output.

Arguments:

name  Name of the model to bake (without the Table suffix). You can use
      Plugin.name to bake plugin models. (optional)

Omitting all arguments and options will list the table names you can
generate models for.

````

We can use option `--no-fixture` to skip the generate / update existing fixture. Sometimes, the adjustment on existing fixtures will caused the trouble in Unit Test (if we write the Unit Test and use the fixtures).